connectivity/FEATURE_BLE/libraries/TARGET_CORDIO_LL/CMakeLists.txt

```
@@ -22,8 +22,8 @@ target_include_directories(mbed-os
         ${CMAKE_CURRENT_SOURCE_DIR}/stack_adaptation
 )
 
-target_sources(mbed-os
-    PRIVATE
+target_sources(mbed-os-ble-cordio
```

```suggestion
target_sources(mbed-os-ble-cordio_ll
```
All the source files mentioned here are under `TARGET_CORDIO_LL`, they should technically belong to the corresponding CMake target.